ras_backstage/controllers/party_controller.py

```diff
@@ -0,0 +1,37 @@
+import json
+import logging
+
+from structlog import wrap_logger
+
+from ras_backstage import app
+from ras_backstage.common.requests_handler import request_handler
+from ras_backstage.exception.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_party_by_business_id(party_id):
+    logger.debug('Retrieving business party', party_id=party_id)
+    url = f'{app.config["RAS_PARTY_SERVICE"]}party-api/v1/businesses/id/{party_id}'
+    response = request_handler('GET', url, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code != 200:
+        logger.error('Error retrieving business party', party_id=party_id)
+        raise ApiError(url, response.status_code)
+
+    logger.debug('Successfully retrieved business party', party_id=party_id)
+    return json.loads(response.text)
+
+
+def get_party_by_respondent_id(party_id):
+    logger.debug('Retrieving respondent party', party_id=party_id)
+    url = f'{app.config["RAS_PARTY_SERVICE"]}party-api/v1/respondents/id/{party_id}'
```

Comment:
Might be worth using `urljoin` on config-prepended strings, depending on how accurate they need to be?

Reply:
I like the idea, this is something we'd probably want to do across a lot of our repos rather than in this particular PR?

Reply:
I think we're wanting to merge this in to move release 13 on right now but we can look into using `urljoin` in the next batch of changes we make in this type of repo

Reply:
Sure thing 👍